src/components/tab-nav/tab-nav.tsx

```diff
@@ -370,9 +385,49 @@ export class TabNav {
   }
 
   get enabledTabTitles(): HTMLCalciteTabTitleElement[] {
-    return filterDirectChildren<HTMLCalciteTabTitleElement>(
+    const directEnabled = filterDirectChildren<HTMLCalciteTabTitleElement>(
       this.el,
       "calcite-tab-title:not([disabled])"
     );
+    return directEnabled.filter((tabTitle) => !tabTitle.closed);
+  }
+
+  handleTabTitleClose(closedTabTitleEl: HTMLCalciteTabTitleElement): void {
```

Comment:
Currently, if you close a tab more than one tab "away" from the currently selected tab, the selected tab is shifted. I would not expect the selected tab to change unless I close it or navigate to another, regardless of distance from selected item.

In this example I'd expect "Bicycles" in the bottom example to remain selected when I close the first tab.

https://github.com/Esri/calcite-components/assets/4733155/896df332-96a6-4a06-928e-16d05cbd839a

Reply:
Similarly - in this example, I close "Bicycles" and the selected tab is shifted to "Aircrafts" - when I think we'd want it to stay on "Watercraft"

https://github.com/Esri/calcite-components/assets/4733155/e57a0f8d-e9b6-4453-b591-7c9a598e0a84

Reply:
Cool thanks. I know why that is happening. Pushing a fix and a corresponding test.

Reply:
Pushed. Let me know if it works now.